prisma/seed.ts

```diff
@@ -101,6 +101,27 @@ async function main() {
       storeId: seller1Store.id,
     },
   });
+
+  // ----------------------
+  // 7. 상품 (Notification 테스트용)
+  // ----------------------
+
+  const topCategory = await prisma.category.findFirst({
+    where: { name: 'TOP' },
+  });
+
+  const testProduct = await prisma.product.create({
+    data: {
```

Comment:
```suggestion
    data: {
    id: 'testProductId',
```

id를 따로 안 넣으면 cuid로 랜덤 생성되기 때문에 http 테스트 할 때 id를 다시 찾아서 넣어야 하더라고요, 특정 id로 생성되게 넣어두면 편할 것 같습니다 !